causing/utils.py

```diff
@@ -7,7 +7,6 @@
 from copy import deepcopy
 import json
 
-import pydot
```

Comment:
If this was the last usage of pydot, we can remove it from the list of dependencies.